### PR TITLE
Basic structure and style for Homescreen and Header

### PR DESCRIPTION
**Pull request by juliakimseohyeon**
Hi team, when you have a moment please take a look at the Homescreen and Header. Thank you 😊

### CODE REVIEW

**Comment by juliakimseohyeon** [comment]
Related to Ticket #21

**Comment by nicmart-dev** [comment]
@juliakimseohyeon great job matching with the mockup!

A few notes:

1. As login will be taken opportunistically, I suggest to hide the login icon behind a feature toggle (or comment out the associated code) and also same for the saved recipes section.
2. small enhancement, you used a logout instead of login icon.
3. Add `.DS_Store` to `.gitignore`

**Review by nicmart-dev — CHANGES_REQUESTED** [review]
@juliakimseohyeon great job matching with the mockup!

A few notes:
1. As login will be taken opportunistically, I suggest to hide the login icon behind a feature toggle (or comment out the associated code) and also same for the saved recipes section.
2. small enhancement, you used a logout instead of login icon.
3. Add `.DS_Store` to `.gitignore`

**Review by nicmart-dev — CHANGES_REQUESTED** [review]
Nice job Julia on footer and change language!

A few comments:
- I still see a few `.DS_Store` files in the PR.
- When changing language, some text is still in English (ie. Hungry Chef and "Show me" button, and I think app name itself should be translated). Also when clicking that button in another language, language reverts back to English
image

**Comment by juliakimseohyeon** [comment]
Hi @nicmart-dev, thanks for pointing those out! I've removed the cached `.DS_Store` files and added a onSubmit function to the form on the home page. This should prevent the language from reverting back to English now.

**Review by nicmart-dev — APPROVED** [review]

File: tailwind.config.js
Comment (nicmart-dev) [inline tailwind.config.js:1]: I think this file at the root should be removed, and only the one in client folder remain.

File: package.json
Comment (nicmart-dev) [inline package.json:1]: I think the changes should be undone, as those dependencies belong in the client folder, not at the root.

**Review by nicmart-dev — COMMENTED** [review]
Some minor comments regarding files at the root.

**Pull request merged**